StardewArchipelago/Locations/CodeInjections/Initializers/VanillaCodeInjectionInitializer.cs

```diff
@@ -27,7 +28,7 @@ public static void Initialize(IMonitor monitor, IModHelper modHelper, Archipelag
             JunimoNoteMenuInjections.Initialize(monitor, modHelper, archipelago, state, locationChecker, bundleReader);
             MineshaftInjections.Initialize(monitor, modHelper, archipelago, locationChecker);
             SkillInjections.Initialize(monitor, modHelper, archipelago, locationChecker);
-            QuestInjections.Initialize(monitor, modHelper, archipelago, locationChecker);
+            QuestInjections.Initialize(monitor, modHelper, archipelago, locationChecker, moddedListsAndDictionaries);

```

Comment:
Hopefully, assuming we go the way suggested in the previous comment, we can save a lot on bloat in here for these that are, essentially, string constants. Sorry if I didn't explain clearly enough what I meant in the previous review, I really didn't expect you to make this whole thing, especially after we talked about how the daisy chains suck